.github/workflows/go-scheduled.yml

```diff
@@ -30,40 +32,22 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-go-
 
-    - name: Get dependencies
-      run: |
-        go get -v -t -d ./...
-        if [ -f Gopkg.toml ]; then
-            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-            dep ensure
-        fi
-
-    - name: Format
-      run: |
-        diff -u <(echo -n) <(gofmt -d .)
-
-    - name: Lint
-      run: |
-        curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.30.0
-        /home/runner/go/bin/golangci-lint run --timeout=5m --skip-dirs=pkg/promql --skip-dirs=pkg/promb
-
     - name: Build
-      run: go build -v ./...
```

Comment:
I am not sure yet, but it looks to me like build could also be removed from periodic job as it doesn't test things which are not in control by maintainers of this repository code.